Sprinkle of Yeezus/Settings.swift

```diff
@@ -11,52 +11,68 @@ import UIKit
 import UserNotifications
 
 class SettingsPage: UIViewController {
```

Comment:
Something we can update in another PR (Pull Request), but classes should following naming convention that mentions what class they are subclassing. For instance, this class is subclassing `UIViewController`, so the name of the class should be something like `SettingsPageViewController` or perhaps even better `SettingsViewController` (because there is such thing as a UIPageViewController)

In addition, files should be named the same as the class which inhabits it. So the file for a `SettingsViewController` class should be called `SettingsViewController.swift`